<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Initial</class>
 <widget class="QWidget" name="Initial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>在线网络教学系统客户端</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/icon.png</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1000</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMin">
       <property name="styleSheet">
        <string notr="true">QPushButton#buttonMin
{
    background-color:transparent;
}

QPushButton#buttonMin:hover
{
    background-color:rgb(255, 0, 0);
}

QPushButton#buttonMin:pressed
{
    background-color:rgb(0, 85, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="styleSheet">
        <string notr="true">QPushButton#buttonClose
{
    background-color:transparent;
}

QPushButton#buttonClose:hover
{
    background-color:rgb(255, 0, 0);
}

QPushButton#buttonClose:pressed
{
    background-color:rgb(0, 85, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>430</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>430</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
       </property>
       <property name="title">
        <string>用户信息</string>
       </property>
       <widget class="QLabel" name="currentTimeLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
        </property>
        <property name="text">
         <string>当前用户：</string>
        </property>
       </widget>
       <widget class="QLabel" name="usernameLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;幼圆&quot;;
</string>
        </property>
        <property name="text">
         <string>正在学习：</string>
        </property>
       </widget>
       <widget class="QLabel" name="lastPointnameLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="quitButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>370</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#quitButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}

QPushButton#quitButton:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
}

QPushButton#quitButton:pressed
{
    color:white;
    background-color:rgb(14 , 135 , 228);
    padding-left:3px;
    padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>退出系统</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
       </property>
       <property name="title">
        <string> 管理模块</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>180</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>幼圆</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>知识建构图</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>幼圆</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>教学模块</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget1">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>380</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>幼圆</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>测试模块</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>幼圆</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>个人信息</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="ClickableLabel" name="knowledgeClickLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="educationsystem.qrc">:/images/1.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="ClickableLabel" name="teachClickLabel">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>60</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="educationsystem.qrc">:/images/2.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="ClickableLabel" name="testClickLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>260</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="educationsystem.qrc">:/images/3.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="ClickableLabel" name="userClickLabel">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>260</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="educationsystem.qrc">:/images/4.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">helper\clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="educationsystem.qrc"/>
 </resources>
 <connections/>
</ui>
